README.md

```diff
@@ -1,30 +1,34 @@
 linter-flake8
 =============
 
-linter-flake8 is a [flake8](https://pypi.python.org/pypi/flake8 provider for [Linter](https://github.com/AtomLinter/Linter).
+linter-flake8 is a [flake8](https://pypi.python.org/pypi/flake8) provider for [Atom-Linter](https://github.com/AtomLinter/Linter).
```

Comment:
The package is just called `Linter`, although the eventual goal is to have it integrated into Atom :wink:.